main.go

```diff
@@ -225,6 +226,8 @@ func main() {
 	r.HandleFunc("/img/{collection}/height/{dimension}/{name}", heightHandler)
 
 	r.HandleFunc("/img/{collection}/{width}x{height}/{name}", resizeHandler)
+	http.Handle("/", r)
 	log.Print("Starting imageservice")
-	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", Conf.port), r))
+	adminz.Init(Conf.port, func() string { return "{ \"a\": 1 }" })
```

Comment:
I'd prefer to pass in the explicit list of kill file paths here, rather than hardcode them in admin -- maybe add a helper that takes base path and varargs ports, returns list of paths (including the `all`) one, then use that helper in the call to Init?